src/ClientWidgetApi.ts

```diff
@@ -453,6 +462,32 @@ export class ClientWidgetApi extends EventEmitter {
         });
     }
 
+    private async handleSendToDevice(request: ISendToDeviceFromWidgetActionRequest): Promise<void> {
```

Comment:
My personal preference is to get rid of the `else` and add `return`s to the if statements, but it probably is just that - a personal preference

Reply:
Well, if it's all the same to you, I'm going to keep it this way :stuck_out_tongue: